kuttl-test.yaml

```diff
@@ -13,6 +13,7 @@
 
 apiVersion: kuttl.dev/v1beta1
 kind: TestSuite
+kindNodeCache: true
```

Comment:
This allows the downloaded images to be cached on the node. This will speed up test case execution.